common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -107,7 +107,7 @@
                 this.boardName = null;
                 this.current_search = '';
                 this.mode = 'all';
-                this.showThreadPreview = options.showThreadPreview;
+                this.showThreadPreview = options.showThreadPreview || true;
```

Comment:
I think this will always be true! Maybe we just don't need as an option anymore.